python/psautohint/ufoFont.py

```diff
@@ -1122,6 +1122,14 @@ def buildGlyphHashValue(self, width, outlineXML, glyphName,
             glyph_hash = hashlib.sha512(data.encode("utf-8")).hexdigest()
         return glyph_hash, dataList
 
+    @staticmethod
+    def _represents_integer(string):
```

Comment:
I don't think this is needed. (See comment above.)

Reply:
I agree it seems overkill, but `isdigit()` is not a viable option.